besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1588,7 +1591,7 @@ private void configure() throws Exception {
                     : SyncMode.FULL);
 
     ethNetworkConfig = updateNetworkConfig(getNetwork());
-
+    unstableContractCacheOptions.initConfig();
```

Comment:
I struggled a lot with were to put this. This is the moment where the value in the Option becomes available to the Configuration.